src/components/auth/register.scss

```diff
@@ -0,0 +1,112 @@
+.container{
```

Comment:
This file has been deleted in master, we can remove it as _register.scss is used for styling